app/src/main/res/layout/fragment_bookshelf.xml

```diff
@@ -3,4 +3,20 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"/>
+    android:layout_height="match_parent">
+
+    <ScrollView
```

Comment:
from the project description I can infer that this page is going to have a lot of scrolling down depending on the amount of books that a user has stored for themselves, so this will be useful to have

Reply:
A LinearLayout within a scroll view can have some unpredictable side effects, and can make components difficult to auto size... as it essentially has infinite scroll.   If there are going to be a lot of books, I'd use a RecyclerView instead.